src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -44,7 +44,7 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.command('check-name', 'check_name_availability')
         g.custom_command('create', 'create_storage_account', min_api='2016-01-01')
         g.command('delete', 'delete', confirmation=True)
-        g.command('show', 'get_properties', exception_handler=g.get_handler_suppress_404())
+        g.generic_show_command('show', 'get_properties')
```

Comment:
Does this represent a breaking change to this command?  If we are including this on one command as a test, it should ensure that the behavior is unchanged.  We don't want to go from swallowing a 404 to failing.  Better to go from failing to failing.  We would need to advertise the behavior change.

Reply:
Nope, the `g.get_handler_suppress_404()` only handles data-plane storage 404 exceptions. 
This is an example of an exception handler that was added but never did its intended job.

This won't break any scripts unless the user checked specifically for an exit code of 1.

Reply:
What would they have previously looked for?

Reply:
sample script from #5006 : 
```az group show $resourceGroupName 1> /dev/null

if [ $? != 0 ]; then
...
fi
```
Most script will check for exit code not being 0 or the presence of output, as error codes can vary.